### PR TITLE
Update CommandBar UI, ResultDisplay UI & default window size

### PR DESCRIPTION
**Pull request by itsme-zeix**
Command bar selected (Default):
image

Command bar unselected:
image

Command Error:
image

Do you guys think I should remove the prompt text `Enter command here...' when the command bar is selected? Currently the behaviour only changes the icon and prompt text to be lighter.

### CODE REVIEW

**Review by FionaQY — APPROVED** [review]
LGTM. Great job on the UI and the neat code.

File: src/main/resources/view/css/ResultDisplay.css
Comment (colinhia) [inline src/main/resources/view/css/ResultDisplay.css:1]: Thanks for the hardwork in extracting out the files for easier maintainability!

File: src/main/java/seedu/address/ui/CommandBox.java [hunk 1]
```diff
@@ -59,7 +77,27 @@ private void handleCommandEntered() {
...
+    }
+
+    /** Sets the command box style to indicate the command box is not focused. */
+    private void setStyleToUnfocused() {
+        commandTextIcon.setIconColor(Color.GREY);
+        if (commandTextArea.getStyleClass().contains(ERROR_STYLE_CLASS)) {
+            return;
+        }
+        commandTextArea.setStyle("-fx-text-fill: grey;");
     }
```
Comment (colinhia) [inline src/main/java/seedu/address/ui/CommandBox.java:101]: Good use of abstraction and alignment to the pre-existing setStyleToDefault method!

**Review by colinhia — APPROVED** [review]
Overall LGTM! Thanks for all the hardwork in restructuring the CSS files and the UI Changes!

**Pull request merged**